parlai/mturk/tasks/acute_eval/example_script.py

```diff
@@ -3,39 +3,44 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-from parlai.mturk.tasks.acute_eval.run import main as run_main, add_args
+from parlai.mturk.tasks.acute_eval.run import AcuteEvaluator, add_args
 
 """
 Example script for running ACUTE-EVAL.
-The only arguments that *must* be modified for this to be run are:
-'dialogs_path':  Path to folder containing logs of all model conversations.
-    Each model should have one file named '<modelname>.jsonl'
-'model_comparisons': List of tuples indicating pairs of models to be compared.
-    The model names here should match the names of files in the folder
-'onboarding_tasks': List of tuples in format (id1, id2, name of matchup) where
-    id1, id2 are conversation ids where id2 is the correct conversation's id
-Help strings for the other arguments can be found in run.py
+The only argument that *must* be modified for this to be run is:
+``pairings_filepath``:  Path to pairings file in the format specified in the README.md
+
+The following args are useful to tweak to fit your specific needs;
+    - ``annotations_per_pair``: A useful arg if you'd like to evaluate a given conversation pair
+                                more than once.
+    - ``pairs_per_matchup``:    Essentially, how many pairs of conversations you would like to evaluate
+    - ``subtasks_per_hit``:     How many comparisons you'd like a turker to complete in one HIT
+
+Help strings for the other arguments can be found in run.py.
 """
 
 
 def set_args():
     args = add_args()
-    # task folder containing pairings file
-    args['task_folder'] = 'parlai/mturk/tasks/acute_eval/example/'
+    # pairings file
+    # args['pairings_filepath'] = 'parlai/mturk/tasks/acute_eval/example/pairings.jsonl'
```

Comment:
we can just delete this? If we intend to have the next line as the default example

Reply:
ahh oops i did not mean to push that change, thanks for catching